web/src/features/Dashboard/components/AppStatus.tsx

```diff
@@ -148,7 +147,7 @@ export default class AppStatus extends Component<Props, State> {
               </>
             )}
             <Link
-              to={`${url}/config/${app?.downstream?.currentVersion?.sequence}`}
```

Comment:
I'm not aware of a scenario where this was _not_ `undefined`, so I just removed it all together. Tested this with both single and multi-app installs and both render the correct link for editing config.